xorca/lib.py

```diff
@@ -413,3 +414,90 @@ def load_xorca_dataset(data_files=None, aux_files=None, decode_cf=True,
         get_all_compatible_chunk_sizes(target_ds_chunks, ds_xorca))
 
     return ds_xorca
+
+
+def load_xorca_dataset_zarr(data_stores=None, aux_files=None, decode_cf=True,
```

Comment:
As I'll explain in more detail below, it could be relatively easy to allow for reading netCDF and Zarr stores within the same function. Then, we could just have a generic `load_xorca_dataset` function which uses the [ask forgiveness not permission](https://stackoverflow.com/a/12265860) to find out how to load things in the list of data_{files,stores} and aux_{files,stores}.

The easiest way to get there would, in my opinion, be to not get rid of the current implementation of `load_xorca_dataset`, but to rename `load_xorca_dataset_zarr` to `load_xorxa_dataset_auto`.